e2e/consul/input/consul_wi.nomad.hcl

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+job "example" {
+
+  group "example" {
+    network {
+      port "db" {
+        to = 5678
+      }
+    }
+
+    task "example" {
+      driver = "docker"
+
+      config {
+        image   = "busybox:1"
+        command = "nc"
+        args    = ["-ll", "-p", "1234", "-e", "/bin/cat"]
```

Comment:
Is this port intentionally not the one for the service?

Reply:
no, accidentally ;)

Reply:
addressed in [b66add3](https://github.com/hashicorp/nomad/pull/19500/commits/b66add3774db497cccde45410832b47ccf29d36c)